<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qt Calculator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="btn_ac">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>âŒ«</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>130</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dot">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>480</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(238, 240, 245);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(228, 231, 237);
 }
QPushButton:pressed {	
	background-color: rgb(248, 248, 248);
	color: rgb(94, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_divide">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(173, 225, 255);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(153, 209, 241);
 }
QPushButton:pressed {
	background-color: rgb(205, 237, 255);
	color: rgb(113, 205, 255);
}</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_multiply">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(173, 225, 255);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(153, 209, 241);
 }
QPushButton:pressed {
	background-color: rgb(205, 237, 255);
	color: rgb(113, 205, 255);
}</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
   <property name="class" stdset="0">
    <string>btn_style1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_subtract">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(173, 225, 255);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(153, 209, 241);
 }
QPushButton:pressed {
	background-color: rgb(205, 237, 255);
	color: rgb(113, 205, 255);
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>62</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(173, 225, 255);
	color: rgb(56, 185, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(153, 209, 241);
 }
QPushButton:pressed {
	background-color: rgb(205, 237, 255);
	color: rgb(113, 205, 255);
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_equal">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>62</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(26, 172, 255);
	color: rgb(255, 255, 255);
	border-radius: 16px
}
 QPushButton:hover {
	background-color: rgb(17, 154, 232);
	color: rgb(232, 232, 232);
 }
QPushButton:pressed {
	background-color: rgb(96, 197, 255);
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="math_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>271</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(138, 138, 138);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="space_label_2">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(66, 66, 66);
text-align:right;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="icon_label_3">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(66, 66, 66);</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="result_label">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>32</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(66, 66, 66);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
